app/assets/stylesheets/views/_browse.scss

```
@@ -67,24 +67,19 @@ $topic-section-border: 1px solid $govuk-border-colour;
 .browse__curated-section {
   border-top: $topic-section-border;
   padding-top: govuk-spacing(3);
-  margin-bottom: govuk-spacing(9);
+  margin-bottom: 70px;
```

If we absolutely have to stray completely outside of the Design System spacing, then this either needs a comment explaining why this is a magic number (hopefully preventing it from being undone in the future) or should be achieved by adding together two of the static spacing functions.

```suggestion
  margin-bottom: 70px; // This needs to be 70px because....
```

or



```suggestion
  margin-bottom: govuk-spacing(6) + govuk-spacing(7);
```


If the spacing scale doesn't suit, then we also need to note it down so we can let the Design System team know - feedback like this really helps them ensure that the Design System is useful and relevant.

---

Good shout. This particular value was actually a bit of a test for now for the designer to see what it would look like if we increased the spacing here (we've got a separate conversation going on in the Trello card about design fixes, my bad for not making it clear here). But I've implemented your suggested way of setting the value and have added a comment in case we decide to keep this spacing.